src/test/scala/com/github/sstone/amqp/ConsumerSpec.scala

```diff
@@ -69,16 +81,17 @@ class ConsumerSpec extends ChannelSpec {
       probe.expectMsg(1 second, ChannelOwner.Connected)
       probe.expectMsg(1 second, ChannelOwner.Connected)
       consumer ! Record(AddBinding(Binding(exchange, queue, "my_key")))
-      val Amqp.Ok(AddBinding(_), _) = receiveOne(1 second)
+      val Amqp.Ok(AddBinding(_), _) = probe.receiveOne(1 second)
 
       val message = "yo!".getBytes
       producer ! Publish(exchange.name, "my_key", message)
       probe.expectMsgClass(1.second, classOf[Delivery])
 
       // crash the consumer's channel
       consumer ! DeclareExchange(ExchangeParameters(name = "foo", passive = true, exchangeType =""))
-      receiveOne(1 second)
+      val Amqp.Error(DeclareExchange(_), _) = probe.receiveOne(1 second)
       probe.expectMsgAllOf(1 second, ChannelOwner.Disconnected, ChannelOwner.Connected)
+      val Ok(AddBinding(Binding(`exchange`, `queue`, "my_key")), Some(_)) = probe.receiveOne(1 second)

```

Comment:
after the channel crashes and reconnects, it plays back the `AddBinding` as well